homeassistant/components/water_heater/recorder.py

```diff
@@ -0,0 +1,12 @@
+"""Integration platform for recorder."""
+from __future__ import annotations
+
+from homeassistant.core import HomeAssistant, callback
+
+from . import ATTR_MAX_TEMP, ATTR_MIN_TEMP, ATTR_OPERATION_LIST
+
+
+@callback
+def exclude_attributes(hass: HomeAssistant) -> set[str]:
+    """Exclude static attributes from being recorded in the database."""
+    return {ATTR_OPERATION_LIST, ATTR_MIN_TEMP, ATTR_MAX_TEMP}
```

Comment:
I  guess we can always include the `Entity.capability_attributes` for all integrations as it's stored in the entity registry.

Reply:
Probably attribution and supported_features as well

Reply:
Looks like `fan`, `humidifer`, `date_time`, `number`, `select`, `siren`, and `vacuum` can all get their `capability_attributes` excluded.  

`sensor` needs to keep `ATTR_STATE_CLASS` since its used for statistics.